File: problem0010.java
import java.math.BigInteger;
import java.util.ArrayList;
import java.util.List;

public class Problem0010 {

    private static List<Integer> primes = new ArrayList<>();
    public static void main(String[] args) {
        int primesBelow = 2000000;

        BigInteger sum = BigInteger.ZERO;

        for(int i = 2; i < primesBelow; i++) {
            if(isPrime(i)) {
                primes.add(i);

                sum = sum.add(BigInteger.valueOf(i));
            }
        }

        System.out.println("Sum: " + sum);
    }

    private static boolean isPrime(int i) {
        boolean isPrime = true;
        
        for(int prime : primes) {
            if(i % prime == 0) {
                isPrime = false;
                break;
            }
        }

        return isPrime;
    }
}

File: python/problem_0010.py

# Problem url: https://projecteuler.net/problem=10

primes = [2]

def is_prime(num: int):
    for prime in primes:
        if num % prime == 0:
            return False
    return True

primes_below = 2000000

for i in range(3, primes_below + 1):
    if is_prime(i):
        primes.append(i)
        print(i)

print('Sum of primes below ', primes_below, ' is: ', sum(primes))